### PR TITLE
Using theme provider in components

### PR DESCRIPTION
**Pull request by dvargasfuertes**

### CODE REVIEW

File: app/_common/Layout.tsx [hunk 1]
```diff
@@ -4,36 +4,52 @@ import { Head as DefaultHead } from "@dvargas92495/ui/dist/components/Document";
 import type { MetaFunction } from "remix";
 
 export const themeProps = {
```
Comment (dvargas92495) [inline app/_common/Layout.tsx:6]: reorganized the colors you defined to be interpretable by material ui

File: app/routes/user.tsx [hunk 2]
```diff
@@ -257,7 +257,8 @@ const ProfileContent = () => {
   });
   return (
     <Box sx={{ maxWidth: "600px" }}>
-      <H1>Setup your fundraising profile</H1>
+      <H1 sx={{color: 'color.grey'}}>Setup your fundraising profile</H1>
+      <H1 sx={{color: 'text.secondary'}}>Setup your fundraising profile</H1>
```
Comment (dvargas92495) [inline app/routes/user.tsx:261]: this is how you could use the theme props defined:
- with the `sx` prop
- through Material's `styled` function, could make `color: text.secondary` for example.

**Review by dvargasfuertes — COMMENTED** [review]
Let me know if this gets us closer to the theme structure you're looking for!

File: app/routes/user.tsx (hunk 2 above)
Comment (dvargas92495) [reply app/routes/user.tsx:261]: Will remove the duplication here before merging

**Pull request merged**